src/mock.js

```diff
@@ -77,14 +77,30 @@ const mock = (refract, options) => {
                   responseBody,
                   responseHeaders
                 };
-                mocks.push(mockRoute(nockOptions));
+
+              if(!routes.hasOwnProperty(resourceUrl)) {
+                routes[resourceUrl] = [];
+              }
+
+                routes[resourceUrl].push(nockOptions);
               });
             });
           });
         });
       });
     });
 
+    // sort URLs by length
+    const urls = Object.keys(routes);
+    urls.sort((a, b) => b.length - a.length);
+
+    // collect mocks
+    const mocks = [];
+    urls.forEach(url => {
+      const resources = routes[url];
+      resources.forEach(resource => {mocks.push(mockRoute(resource))})
```

Comment:
You could drop curly braces here

```suggestion
      resources.forEach(resource => mocks.push(mockRoute(resource)))
```

Reply:
Fixed.